Enterprise-CMCS/macpro-mdct-mfp
# WP - Initiatives Details - Close out autosave + autopopulate field

### Description

This ticket will cover three things: 
- auto-populating a disabled field in the close out page
- auto-saving fields
- updating ids to be readable + match with the correct parent

This ticket will NOT cover:
- autosave for nested fields is a known, unresolved bug in MCR and MFP, so that will not work as expected
### Related ticket(s)


MDCT-3026

---
### How to test


- Login
- Go to the Define Initiatives page and fill out the fields, the necessary field to be filled out is the Projected End Date at the bottom. Confirm that autosave is working on this page
- Go to the Close Out page and confirm that the disabled field has autopopulated correctly
- Confirm that fields are autosaving (as of right now, nested fields cannot autosave due to a bug in MCR + MFP)

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
